frontends/concrete-python/requirements.extra-full.txt

```diff
@@ -1,3 +1,6 @@
 matplotlib>=3.7
 pillow>=10.2
 pygraphviz>=1.11
+linkcheckmd>=1.4.0
+linkchecker>=10.3.0
+mistletoe>=0.9.0
```

Comment:
These should be put into dev, requirements, extra is for actual optional deployment dependencies.

Reply:
ok done